docs/developer-docs/daos/nns/concepts/neurons/neuron-overview.mdx

```diff
@@ -46,34 +46,49 @@ In addition, each neuron has an associated **account** on the ICP ledger where t
 ### Neuron permissions
 The **controller** is a principal that controls the neuron and which cannot be changed. A principal corresponds to the public key of a public-private key pair, and anyone with possession of the corresponding private key will have full control of the neuron. Therefore, the private key of the controller principal should be kept very secure. A principal can control many neurons.
 
-In addition to the controller, a neuron can have **hotkeys**. Hotkeys are defined by a list of principal ID and can be used to perform actions with limited privileges, namely voting, following (see below), and reading the full information about the neuron.
+### Hotkeys
+In addition to the controller, a neuron can have **hotkeys**. Hotkeys are principal IDs that can perform actions with limited privileges, namely:
+
+- Voting.
+
+- Following (see below).
+
+- Reading the full information about the neuron, including non-public fields.
+
+Hotkeys cannot increase dissolve delay, start dissolving, or disburse a neuron's stake.
+
+A neuron can have multiple hotkeys, with at most 15 hotkeys per topic on a neuron. One common workflow using hotkeys is to set the primary controller of the neuron as a hardware wallet, then vote or view neuron information through a software wallet set as a hotkey.
```

Comment:
> set the primary controller of the neuron as a hardware wallet

An ICP neuron can only have 1 controller. There is no concept of "primary controller". I believe, SNS neurons can have multiple controllers, but there is also no distinction between them.

Also, I would say the hardware wallet is set as the controller, rather than the controller set as the hardware wallet.

Reply:
Revised per feedback